idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/mode/iconpack/conversion/IconPackConversionState.kt

```diff
@@ -9,6 +8,7 @@ sealed interface IconPackConversionState {
 
     data class BatchFilesProcessing(
         val iconsToProcess: List<BatchIcon> = emptyList(),
+        val showLoading: Boolean = false,
```

Comment:
If we can add a new class state named `Loading` or something like that? It would be more clear.

Reply:
Agree 👍